neurokit2/hrv/intervals_process.py

```diff
@@ -102,6 +108,12 @@ def intervals_process(
         )
 
         intervals = signal_interpolate(intervals_time, intervals, x_new=x_new, **kwargs)
+    else:
+        # check if intervals appear to be already interpolated
+        if _intervals_time_uniform(intervals_time):
+            # get sampling rate used for interpolation
+            interpolation_rate = _intervals_time_to_sampling_rate(intervals_time)
+
     if detrend_method is not None:
```

Comment:
perhaps i would have this argument named just `detrend = None` (for no detrending) or a string with a specific method name?